packages/regl-worldview/src/Worldview.js

```diff
@@ -262,24 +262,35 @@ export class WorldviewBase extends React.Component<BaseProps, State> {
   }
 
   render() {
-    const { width, height, showDebug, keyMap, shiftKeys, style } = this.props;
+    const { width, height, showDebug, keyMap, shiftKeys, style, cameraState } = this.props;
     const { worldviewContext } = this.state;
+    const canvasHtml = [
```

Comment:
#nit If you don't want to have to bother with `key` you could use `const canvasHtml = <React.Fragment>`

Reply:
Good idea! I actually tried that first, but was getting some weird lint?/flow? error so I gave up. Now it's all gone, weird.